docs/user_guide/tutorials/02-collaboration/index.md

```diff
@@ -0,0 +1,221 @@
+# Collaborative Features in JupyterGIS
+
+Welcome to the JupyterGIS collaborative features tutorial. JupyterGIS enables seamless sharing of notebooks and GIS files, allowing teams—including GIS specialists, data analysts, and other experts—to work together on spatial projects in a shared environment. This guide will provide you with the tools and steps needed to collaborate effectively, using features like real-time editing, cursor tracking, follow mode, and annotations.
+
+## Motivation
+
+Collaboration is at the heart of effective GIS projects. Teams often include members with diverse backgrounds, skills, and areas of expertise. Without robust collaborative tools, it can become challenging to share insights, make real-time decisions, and maintain consistency across project workflows. JupyterGIS simplifies collaboration by providing real-time editing, annotations, and interactive features that allow teams to seamlessly integrate their work.
+
+:::{admonition} Objectives
+:class: seealso
+By following this tutorial, you will be able to:
+
+- Generate shareable links to invite collaborators.
+- Work together on GIS files and notebooks with live updates.
+- Use follow mode to monitor collaborator activities.
+- Add annotations and comments to provide context, ask questions, or share insights.
+  :::
+
+:::{admonition} Prerequisites
+:class: warning
+Before beginning this tutorial, JupyterGIS must be installed on your computer (see [Installation instructions](https://jupytergis.readthedocs.io/en/latest/user_guide/install.html)) or you can use an online version of JupyterGIS. Currently, real-time collaboration is not supported in [JupyterLite](https://jupytergis.readthedocs.io/en/latest/lite/lab/index.html?path=france_hiking.jGIS/).
+:::
+
+---
+
+## 1. Sharing Your Document
+
+### 1.1. Create Your GIS File
+
+- If you are using a local installation, run JupyterLab.
+
+```
+jupyter lab
+```
+
+- If you are using an online version, navigate to the JupyterGIS platform.
+- In the JupyterLab Launcher, scroll down to the **Other** section.
+- Click **GIS File** to open a blank canvas for your project.
+
+![New GIS File](images/new_gis_file.png)
```

Comment:
```suggestion
  ![New GIS File](images/new_gis_file.png)
```

The indentation tells the documentation generator to treat the image as if it's part of a bullet. As it is now, the images break up the bulleted list.